products/logging/terraform.yaml

```diff
@@ -14,7 +14,7 @@
 overrides: !ruby/object:Overrides::ResourceOverrides
   Metric: !ruby/object:Overrides::Terraform::ResourceOverride
     id_format: "{{name}}"
-    import_format: ["{{name}}"]
+    import_format: ["{{name}}", "{{project}} {{name}}"]
```

Comment:
Should this be {{project}}/{{name}}?

Reply:
Unfortunately no, `{{name}}` can contain slashes.  :(